spark-template-pebble/src/test/java/spark/template/pebble/PebbleExample.java

```diff
@@ -17,9 +17,7 @@ public static void main(String[] args) {
 			Map<String, Object> attributes = new HashMap<>();
 			attributes.put("message", "Hello World!");
 
-			// The hello.pebble file is located in directory:
-			// src/test/resources/spark/template/pebble
-			return new ModelAndView(attributes, "templates/hello.pebble");
+			return new ModelAndView(attributes, "src/test/resources/templates/hello.pebble");
```

Comment:
It seems like both src/test/resources/templates/hello.pebble and templates/hello.pebble work, but I'd be lying if I knew why. Nevertheless, looks like the old path is fine.